src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (a) => {
+    const temporarySum = sum;
```

Comment:
You can put this assignment inside `if` to reduce number of operations.

Reply:
Thanx, great advice ))